src/model/timestamp.rs

```diff
@@ -282,6 +282,18 @@ impl From<&Timestamp> for Timestamp {
     }
 }
 
+impl PartialOrd for Timestamp {
+    fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
+        Some(self.unix_timestamp().cmp(&other.unix_timestamp()))
```

Comment:
Since the implementation is the same, we can implement `PartialOrd` in terms of `Ord`:
```suggestion
        Some(self.cmp(other))
```